randovania/data/json_data/prime1.txt

```diff
@@ -2077,54 +2177,106 @@ Asset id: 3180103205
 > Dock to Transport Tunnel B; Heals? False
   > Transport Tunnel B/Dock to Root Cave
       Trivial
+  > Root Cave/Dock to Root Tunnel
+      Missile
+  > Root Cave/Crossed grapple beam gap
+      Grapple Beam
 
 > Dock to Root Tunnel; Heals? False
   > Root Tunnel/Dock to Root Cave
       Trivial
+  > Root Cave/Dock to Transport Tunnel B
+      Trivial
+  > Root Cave/Crossed grapple beam gap
+      Grapple Beam
 
 > Dock to Arbor Chamber; Heals? False
   > Arbor Chamber/Dock to Root Cave
       Trivial
+  > Root Cave/Pickup (Missile Expansion)
+      X-Ray Visor and Space Jump Boots
+  > Root Cave/Crossed grapple beam gap
+      Trivial
+
+> Pickup (Missile Expansion); Heals? False
+  > Root Cave/Dock to Arbor Chamber
+      X-Ray Visor and Space Jump Boots
+  > Root Cave/Crossed grapple beam gap
+      Trivial
+
+> Crossed grapple beam gap; Heals? False
+  > Root Cave/Dock to Transport Tunnel B
+      Trivial
+  > Root Cave/Dock to Root Tunnel
+      Missile
+  > Root Cave/Dock to Arbor Chamber
+      X-Ray Visor and Space Jump Boots
+  > Root Cave/Pickup (Missile Expansion)
+      X-Ray Visor and Space Jump Boots
 
 ----------------
 Artifact Temple
 Asset id: 597223686
 > Dock to Temple Lobby; Heals? False
   > Temple Lobby/Dock to Artifact Temple
       Trivial
+  > Artifact Temple/Pickup (Artifact of Truth)
+      Trivial
+
+> Pickup (Artifact of Truth); Heals? False
+  > Artifact Temple/Dock to Temple Lobby
+      Trivial
```

Comment:
Missing nodes regarding Meta Ridley and using the teleporter. Add two nodes:

- "Event - Meta Ridley" with the requirements of having every artifact + requirements for fighting Meta Ridley (this also needs to have events added for bosses, flags, etc so this can come later if you want.)
- "Teleport to Impact Crater - Crater Impact Point" with the requirement being "After Meta Ridley" (can add later after event is added)

Also, have door connect to pickup and event, and Ridley to Teleporter